bin/001-import.js

```diff
@@ -21,23 +21,21 @@ let options = {
 	raven: raven, 
 }
 async.parallel({
-	campground: async.apply(helper.create, options, Campground),
-	reservation: async.apply(helper.create, options, Reservation),
+	campground: async.apply(helper.create, options, Campground, function(err, campground){helper.create(options, Reservation)}),	
```

Comment:
I think this line shoudn't work. or it does?
interesting solution, but it's a bit confusing

Reply:
As per the comment I added on my code, the code breaks when trying to send an email.  This was an attempt to fix it.  Because I thought because of the async operation, the reservation is maybe created before the campground and thats why it picks up campground as null.  So I added the reservation as a callback function in the campground.  But this is not the solution.  The code still breaks.

Reply:
Got it. yeah, this is not a solution. i assume main trouble are - we need to setup email config. and all will work fine for you. btw, where do you get that configuration? i mean do you sign in at sendgrid or mailgun and get a propper email server?

Reply:
I got the config from the tutorial and just replaced their details with my own details. I created an app password in Gmail, which I removed when I submitted the code. Will have a look at sendgrid and mailgun.